csrc/multidevice/communication.h

```diff
@@ -81,8 +90,21 @@ class Communication : public Expr {
     return attribute<CommunicationType>(0);
   }
 
-  const DeviceMesh& mesh() const {
-    return attribute<DeviceMesh>(1);
+  TensorView* out() const {
```

Comment:
`out`, `in` will throw on instances created with the constructor without I/O Tvs, right?